### PR TITLE
3rd-party: using "repository" for headers

### PR DESCRIPTION
**Pull request by castulo**
Headers are printed at the beginning of the output for every repository
in operations that run in multipl repos. This header was using the word
"repo" but we are trying to use the whole "repository" word every time
we printed it to the user. This commit changes that.

Signed-off-by: Castulo Martinez <castulo.martinez@intel.com>

### CODE REVIEW

File: test/functional/3rd-party/3rd-party-bundle-list-basic.bats [hunk 1]
```diff
@@ -30,9 +30,9 @@ global_setup() {
 
 	assert_status_is "$SWUPD_OK"
 	expected_output=$(cat <<-EOM
-		_______________________
```
Comment (otaviobp) [inline test/functional/3rd-party/3rd-party-bundle-list-basic.bats:33]: Isn't it better to filter this from the output so we don't need to add that to tests?

Comment (otaviobp) [reply test/functional/3rd-party/3rd-party-bundle-list-basic.bats:33]: And in json_output the output is not very cool. We print one `_` per line

Comment (castulo) [reply test/functional/3rd-party/3rd-party-bundle-list-basic.bats:33]: > Isn't it better to filter this from the output so we don't need to add that to tests?

yeah, that may be a good idea. We can always use `--identical` for those tests we really want to check all output.

Comment (castulo) [reply test/functional/3rd-party/3rd-party-bundle-list-basic.bats:33]: > And in json_output the output is not very cool. We print one `_` per line

yes, it is not cool, but I didn't know how to fix this, currently there is no way of making exceptions of what we include in the json output. I am open to suggestions @otaviobp

Comment (otaviobp) [reply test/functional/3rd-party/3rd-party-bundle-list-basic.bats:33]: I made a PR trying to fix that: #1510

Comment (castulo) [reply test/functional/3rd-party/3rd-party-bundle-list-basic.bats:33]: +1 ☝️

**Comment by castulo** [comment]
Rebased

**Pull request merged**